docs/runbook/README.md

```diff
@@ -6,15 +6,14 @@ This page gathers the available guides to operate a Mithril network.
 
 # Guides
 
-| Operation | Location | Description
-|------------|------------|------------
-| **Genesis manually** | [manual-genesis](./genesis-manually/README.md) | Proceed to manual (re)genesis of the aggregator certificate chain.
-| **Era markers** | [era-markers](./era-markers/README.md) | Create and update era markers on the Cardano chain.
-| **Downloads statistics** | [downloads statistics](./statistics/README.md) | Display the number of downloads per day.
-| **Signer registrations monitoring** | [registrations-monitoring](./registrations-monitoring/README.md) | Gather aggregated data about signer registrations (versions, stake, ...).
-| **Update protocol parameters** | [protocol-parameters](./protocol-parameters/README.md) | Update the protocol parameters of a Mithril network.
-| **Recompute certificates hash** | [recompute-certificates-hash](./recompute-certificates-hash/README.md) | Recompute the certificates has of an aggregator.
-| **Fix terraform lock** | [terraform-lock](./terraform-lock/README.md) | Fix a terraform lock in CD workflows.
-| **Manage SSH access to infrastructure** | [ssh-access](./ssh-access/README.md) | Manage SSH access on the VM of the infrastructure for a user.
-
-
+| Operation                               | Location                                                               | Description                                                               |
+| --------------------------------------- | ---------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| **Genesis manually**                    | [manual-genesis](./genesis-manually/README.md)                         | Proceed to manual (re)genesis of the aggregator certificate chain.        |
+| **Era markers**                         | [era-markers](./era-markers/README.md)                                 | Create and update era markers on the Cardano chain.                       |
+| **Downloads statistics**                | [downloads statistics](./statistics/README.md)                         | Display the number of downloads per day.                                  |
+| **Signer registrations monitoring**     | [registrations-monitoring](./registrations-monitoring/README.md)       | Gather aggregated data about signer registrations (versions, stake, ...). |
+| **Update protocol parameters**          | [protocol-parameters](./protocol-parameters/README.md)                 | Update the protocol parameters of a Mithril network.                      |
+| **Recompute certificates hash**         | [recompute-certificates-hash](./recompute-certificates-hash/README.md) | Recompute the certificates has of an aggregator.                          |
```

Comment:
This should not be:
```suggestion
| **Recompute certificates hash**         | [recompute-certificates-hash](./recompute-certificates-hash/README.md) | Recompute the certificates of an aggregator.                          |
```

Reply:
Good catch, maybe more a typo on `hash`:
```suggestion
| **Recompute certificates hash**         | [recompute-certificates-hash](./recompute-certificates-hash/README.md) | Recompute the certificates hash of an aggregator.                          |
```

Reply:
Done :heavy_check_mark:

Reply:
Yes, that's better!